src/stages/glue-transform.ts

```diff
@@ -0,0 +1,68 @@
+import * as glueAlpha from '@aws-cdk/aws-glue-alpha';
+import * as glue from 'aws-cdk-lib/aws-glue';
+import * as sfn from 'aws-cdk-lib/aws-stepfunctions';
+import * as tasks from 'aws-cdk-lib/aws-stepfunctions-tasks';
+import { Construct } from 'constructs';
+import { StateMachineStage, StateMachineStageProps } from '../pipelines/stage';
+
+export interface GlueTransformStageProps extends StateMachineStageProps {
+  readonly job?: glueAlpha.Job;
```

Comment:
You'll notice I am only providing one "escape hatch" for the `job`, but not a separate one for the `props` as was done in other PRs. The reason is that I don't think we are gaining much (2 LOC really):
```
GlueTransformStage(
    ...,
    job=glue.Job(
        // job parameters here
    ),
)
```
vs
```
GlueTransformStage(
    ...,
    jobProps: {
        // job parameters here
    }
)
```

Reply:
Yeah I guess this job is quite different as it doesn't actually create the Glue job, it merely receives a reference to an existing job. Same goes with the crawler. Since we're only really using the job and crawler names, I don't know if we even need to pass the Job/Crawler, rather than just the name.

Reply:
Removed the job & the crawler keeping just their names